src/ListGroupItem.js

```diff
@@ -1,100 +1,48 @@
-import React, { cloneElement } from 'react';
+import React from 'react';
+import OldListGroupItem from './deprecated/ListGroupItem';
 import bootstrapUtils, { bsStyles, bsClass } from './utils/bootstrapUtils';
+import deprecated from 'react-prop-types/lib/deprecated';
+
 import { State } from './styleMaps';
 import classNames from 'classnames';
 
 class ListGroupItem extends React.Component {
 
-
   render() {
-    let classes = bootstrapUtils.getClassSet(this.props);
+    let { componentClass, header } = this.props;
 
-    classes.active = this.props.active;
-    classes.disabled = this.props.disabled;
-
-    if (this.props.href) {
-      return this.renderAnchor(classes);
-    } else if (this.props.onClick) {
-      return this.renderButton(classes);
-    } else if (this.props.listItem) {
-      return this.renderLi(classes);
+    if (header) {
+      return <OldListGroupItem {...this.props} />;
```

Comment:
So I guess my question, is it alright that I remove the previous `header` prop behavior? It will be a breaking change.

- [ ] Yes (Tyler will update the PR)
- [ ] No (Tyler will update the PR)

:tada: